module/reports/helper.js

```diff
@@ -170,27 +115,16 @@ module.exports = class ReportsHelper {
 						},
 					})
 				}
-				// }
 
 				await Promise.all(
-					projectDetails.map(async function (project) {
+					projectDetails.map(async (project) => {
 						if (project.categories) {
 							project.categories.map((category) => {
-								if (category.externalId !== '' && categories[category.externalId]) {
-									categories[category.externalId] = categories[category.externalId] + 1
-								} else if (categories[category.name]) {
-									categories[category.name] = categories[category.name] + 1
-								} else {
-									if (category.externalId !== '') {
-										categories[category.externalId] = 1
-									} else {
-										categories[category.name] = 1
-									}
-								}
+								let key = category.externalId || category.name
```

Comment:
@MallanagoudaB why this logic change. what happens if else part of old logic come in place?